clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java

```diff
@@ -316,13 +316,13 @@ private void seekBlockInStream(long newPos) throws IOException {
           mCurrentCacheStream =
               mContext.getTachyonBlockStore().getOutStream(currentBlockId, blockSize, address);
         } catch (IOException ioe) {
-          LOG.warn("Failed to write to TachyonStore stream, block {} will not be in "
+          LOG.warn("Failed to write on TachyonStore stream, the blockId {} will not be in "
```

Comment:
this is the same message as the other two "the blockId {} will not be in TachyonStorage" messages, lets refactor the string out